Assets/Scripts/2D/StartGuiManagerScript.cs

```diff
@@ -190,6 +203,7 @@ public void PostProgressOp_LoadAction()
         _postProgressOp -= PostProgressOp_LoadAction;
     }
 
+    /// <summary>Starts to load the heightmap image.</summary>
```

Comment:
I would change this to "Initializes the The 'Load File' dialog for heightmap loading"